src/components/redux/configStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import dataReducer from './actions';
+import coordReducer from './coord';
+
+const allReducers = combineReducers({
+  data: dataReducer,
+  coord: coordReducer,
+});
+
+const store = createStore(allReducers, applyMiddleware(thunk));
+
+export default store;
```

Comment:
## [OPTIONAL]

- [ ] To structure well your project I would really suggest you move the `/redux` directory outside the `/components` directory, as redux only cares about the store and not components. This structure example would help :point_down: 

```
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /container
    |- /pages
    |- /redux
  |- /node_modules
```

Reply:
okay